actionview/lib/action_view/railtie.rb

```diff
@@ -46,13 +45,6 @@ class Railtie < Rails::Engine # :nodoc:
       end
     end
 
-    initializer "action_view.finalize_compiled_template_methods" do |app|
-      ActiveSupport.on_load(:action_view) do
-        ActionView::Template.finalize_compiled_template_methods =
-          app.config.action_view.delete(:finalize_compiled_template_methods)
-      end
-    end
-
```

Comment:
Need a deprecation-like message saying the config no longer does anything?

Reply:
Deprecated in 88b3c454f753c0ce85e8df01d5aacf70fcf29b91